Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
You can get the same thing, cross platform with

```json
"scripts": {
  "clean": "rimraf node_modules",
  "test": "mytestcommand"
}
```

and `npm run clean && npm i && npm test`. If you want to have it even shorter, just add an alias to scripts 

```json
"make": "npm run clean && npm install && npm test"
```